### PR TITLE
ansible: Add proxy documentation

### PR DESCRIPTION
**Pull request by erick0z**
Add a section called proxy in a separate file (proxy.rst). This section should provide the necessary information to deploy CIAO when running behind a proxy.

### CODE REVIEW

File: examples/ciao/proxy.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+=====
+
+If you are running behind a proxy, the following additional steps are needed:
+This applies to Ubuntu and Fedora.
+
+In the deployment node
+----------------------
+
+Make sure apt can run. Edit ``/etc/apt/apt.conf`` (to install ansible and 
+dependencies), appending it with::
```
Comment (leoswaldo) [inline examples/ciao/proxy.rst:11]: I would say something like  "append next line and replace the corresponding values of `<your-proxy>` and `<proxy-port>`:

File: examples/ciao/proxy.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  Acquire::http::Proxy "http://yourproxyaddress:proxyport";
```
Comment (leoswaldo) [inline examples/ciao/proxy.rst:13]: just a comment, I would replace `http://yourproxyaddress:proxyport`  with `http://<your-proxy>:<proxy-port>`  to make more emphasis they are variables that need to be changed

**Comment by erick0z** [comment]
This will be reformat in markdown. Closing this PR

**Comment by albertomurillo** [comment]
1) Make this file a .md rather than .rst for consistency
2) Change commit message to start with ciao: <description>
3) Add "Fixes #47" at the end of the commit message

**Pull request closed without merging**